config/settings.yml

```diff
@@ -29,7 +29,7 @@ CAP:
 PUBMED:
   LOG: log/pubmed.log
   FETCH_PATH: /entrez/eutils/efetch.fcgi?db=pubmed&retmode=xml
-  HOST: eutils.ncbi.nlm.nih.gov
+  HOST: https://eutils.ncbi.nlm.nih.gov
```

Comment:
Sorry to be so pendantic, but now I'm wondering whether this should be `PUBMED.BASE_URL` to distinguish it from the other settings that do not have the protocol in the `.HOST` values?  Another option is to separate out the protocol, as in:
```ruby
"#{PUBMED.PROTOCOL}#{PUBMED.HOST}"
```
If the current solution is sufficient, go with it.  Just offering these thoughts rather than let them drift into the ether.  If you want to keep the current solution, we can merge as is (everything else looks good).